mlbench_core/evaluation/goals.py

```diff
@@ -87,66 +87,25 @@ def task2_time_to_accuracy_light_goal():
     return time_to_accuracy_goal(80)
 
 
-def task3_time_to_preplexity_goal(metric_name, value, tracker):
-    """Time to perplexity goal for benchmark task 3: Language Modelling
+def task3_time_to_perplexity_goal(threshold=70):
```

Comment:
did you change it from 50 to 70? still consistent with the task descr?

Reply:
50 is way to low for such a model to reach it. Here's a score sheet for different models. https://paperswithcode.com/sota/language-modelling-on-wikitext-2.

I have modified the task description accordingly

Reply:
thanks, looks good then